extension/src/time_weighted_average.rs

```diff
@@ -401,6 +402,25 @@ pub fn arrow_time_weighted_average_average(
     time_weighted_average_average(sketch)
 }
 
+mod toolkit_experimental {
+    use super::*;
+    use crate::accessors::toolkit_experimental::*;
+
+    #[pg_operator(immutable, parallel_safe)]
+    #[opname(->)]
+    pub fn arrow_time_weighted_average_integral(
+        tws: Option<TimeWeightSummary>,
+        accessor: AccessorIntegral,
+    ) -> Option<f64> {
+        time_weighted_average_integral(
+            tws,
+            String::from_utf8_lossy(accessor.bytes.as_slice()).to_string(),
+        )
+    }
+}
+
+pub use toolkit_experimental::*;
```

Comment:
I *think* that we've typically said that arrow functions are considered experimental as long as one of its arguments is experimental, i.e. you might not need the surrounding `mod toolkit_experimental {}`